marimo/_output/formatters/panel_formatters.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Marimo. All rights reserved.
+from __future__ import annotations
+
+from typing import Any
+
+from marimo._messaging.mimetypes import KnownMimeType
+from marimo._output.formatters.formatter_factory import FormatterFactory
+from marimo._plugins.ui._impl.from_panel import from_panel
+
+
+class PanelFormatter(FormatterFactory):
+    @staticmethod
+    def package_name() -> str:
+        return "panel"
+
+    def register(self) -> None:
+        import panel  # type: ignore [import-not-found]
+        import param
```

Comment:
you _could_ do `pm.extension()` if there is benefit is that being always included (instead of the user having to do it)

Reply:
Will aim for consistency with other environments (technically the extension call already isn't really required now unless you need some extension to be loaded, e.g. `pn.extension('tabulator')`).